scss/home.scss

```diff
@@ -0,0 +1,254 @@
+
```

Comment:
go through this file and make sure that css chunks are appropriately indented. they should all look like you `.header` chunk

Reply:
Resolved